salt/pillar/http_yaml.py

```diff
@@ -67,12 +66,15 @@ def __virtual__():
 def ext_pillar(minion_id,
                pillar,  # pylint: disable=W0613
                url,
-               with_grains=False):
+               with_grains=False,
+               **kwargs):
```

Comment:
I would prefer to accept `username` and `password` instead of adding `**kwargs`.